package.json

```diff
@@ -12,6 +12,7 @@
     "cors": "^2.4.1",
     "csv-write-stream": "^0.1.2",
     "ddos": "",
+    "dns": "^0.2.2",
```

Comment:
Thanks for the excellent point.  This line in package.json isn't necessary.  I'll delete it.